workers/tasks/export/metadata.ts

```diff
@@ -68,6 +70,10 @@ export const getPubMetadata = async (pubId) => {
 		citationStyle: pubData.citationStyle,
 		citationInlineStyle: pubData.citationInlineStyle,
 		nodeLabels: pubData.nodeLabels,
+		...(primaryCollection?.kind === 'issue' && { citeAs: pubData.community.citeAs }),
```

Comment:
We might not need this line anymore If we implement my suggested changes to the frontmatter render functions.